README.md

```diff
@@ -372,6 +372,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [DeepDive](http://deepdive.stanford.edu) - Creates structured information from unstructured data and integrates it into an existing database.
 * [Deeplearning4j](http://deeplearning4j.org/) - Distributed and multi-threaded deep learning library.
 * [H2O](http://0xdata.com/) - Analytics engine for statistics over big data.
+* [JSAT](https://github.com/EdwardRaff/JSAT) -  Numerous Machine Learning algorithms for pre-processing, classification, regression, and clustering. Some algorithms support mult-threaded execution. 
```

Comment:
One whitespace too many. Also several spelling errors, what do you think of changing the description to:

"Algorithms for pre-processing, classification, regression, and clustering with support for multi-threaded execution."

Machine learning is implied since the entry is in the machine learning section.